cmd/backup-manager/app/constants/constants.go

```diff
@@ -64,5 +64,5 @@ const (
 	ServiceAccountCAPath = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
 
 	// DefaultTableRegex is the default regular expression for 'db.table' matching
-	DefaultTableRegex = "^(?!(mysql|test|INFORMATION_SCHEMA|PERFORMANCE_SCHEMA|METRICS_SCHEMA|INSPECTION_SCHEMA))"
+	DefaultTableRegex = "^(?!(mysql|INFORMATION_SCHEMA|PERFORMANCE_SCHEMA|METRICS_SCHEMA|INSPECTION_SCHEMA))"
```

Comment:
@onlymellb As our e2e test hard code using `test` to insert data, I suggest remove `test` from DefaultTableRegex. WDYT.

Reply:
When using mydumper to back up, users do not need to back up the test DB by default, so I think it is better to create a new database in e2e to insert data.